sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/ConnectionString/ConnectionStringParser.cs

```diff
@@ -92,6 +93,44 @@ internal static string GetIngestionEndpoint(this AzureCoreConnectionString conne
             return uri.AbsoluteUri;
         }
 
+        /// <summary>
+        /// Evaluate connection string and return the requested endpoint.
+        /// </summary>
+        /// <remarks>
+        /// Parsing the connection string MUST follow these rules:
+        ///     1. check for explicit endpoint (location is ignored)
+        ///     2. check for endpoint suffix (location is optional)
+        ///     3. use default endpoint (location is ignored)
+        /// This behavior is required by the Connection String Specification.
+        /// </remarks>
+        internal static string GetLiveEndpoint(this AzureCoreConnectionString connectionString)
```

Comment:
Can we repurpose `GetIngestionEndpoint`, this looks identical copy of it.

Reply:
done.